src/main/scala/com/databricks/labs/overwatch/env/Workspace.scala

```diff
@@ -1,7 +1,7 @@
 package com.databricks.labs.overwatch.env
 
 import com.databricks.dbutils_v1.DBUtilsHolder.dbutils
-import com.databricks.labs.overwatch.ApiCall
+import com.databricks.labs.overwatch.{ApiCall, ApiCallV2}
```

Comment:
```suggestion
import com.databricks.labs.overwatch.{ApiCallV2}
```

We should be able to remove ApiCall from the imports once V2 is fully implemented.